packages/stops-overlay/src/StopsOverlay.story.js

```diff
@@ -44,23 +45,33 @@ class Example extends Component {
     };
   }
 
+  componentDidMount = () => {
+    const { stops } = this.props;
+    this.setState({ stops });
+  };
+
   refreshStops = bounds => {
-    const stops = mockStops.filter(
+    const { stops, filterStops } = this.props;
+    const filteredStops = stops.filter(
```

Comment:
This is probably fine for now, but I think that eventually this will need to change to a test of whether the map bounds includes a portion of a flex zone. It is a more complicated test but would allow to remove the `filterStop` prop.

Reply:
I think this method is only present to avoid performance issues when loading in all of the trimet test data. I don't think we've ever needed it outside this story!